### DISCUSSION TITLE
Typescript Package

### DISCUSSION THREAD

**Discussion by rossparachute** [dojo-engineering/feedback#13] [General]
Hey folks,

Is there a typescript package with typings for the payment component configs etc.?

Thanks,
Ross

**Comment by martinhay1** [comment]
Hi Ross,
Thanks for your query. At present, we don't have Typescript examples available, but you can reference the JavaScript examples we provide here: https://docs.dojo.tech/payments/accept-payments/components/card
I understand this may not be ideal, and we appreciate your feedback. We're always working to improve our developer resources, and I'll forward your request for TypeScript examples to our team.
Thanks,
Martin

**Comment by rossparachute** [comment]
Nah, no problem at all! Just a nice-to-have, appreciate it can mean a fair
amount of refactoring if it's not there already.

Have a good one,
Ross

On Thu, Aug 3, 2023 at 9:54 AM Martin Hay ***@***.***> wrote:

> Hi Ross,
> Thanks for your query. At present, we don't have Typescript examples
> available, but you can reference the JavaScript examples we provide here:
> https://docs.dojo.tech/payments/accept-payments/components/card
> I understand this may not be ideal, and we appreciate your feedback. We're
> always working to improve our developer resources, and I'll forward your
> request for TypeScript examples to our team.
> Thanks,
> Martin
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/dojo-engineering/discussions/13#discussioncomment-6625394>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQF745JJRVCCMAGXRPSHOBDXTNRKVANCNFSM6AAAAAA27YOIMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 


Ross Hendry

Director of Software

t. 0141 846 1765

***@***.*** | www.parachute.net <https://www.thisisparachute.com/>

<https://www.facebook.com/ThisIsParachute/>
<https://twitter.com/ThisIsParachute>
<https://www.instagram.com/thisisparachute/?hl=en>
<https://www.linkedin.com/company/parachute-digital-solutions-limited/>
***@***.***,-4.2789445,17z/data=!3m1!4b1!4m5!3m4!1s0x48884432181689e7:0x4857727e622e8aea!8m2!3d55.8741653!4d-4.2767505>

Confidentiality Note: This email may contain confidential and/or private
information.

If you received this email in error please delete and notify sender.
